README.md

```diff
@@ -1,2 +1,119 @@
-# exposure-notifications-metrics
-Aggregate EN metrics into one place
+# COVID-19 Exposure Notification Apps - Metrics Aggregation Tools
+
```

Comment:
Add the following attribution in normal print + maybe in italic

Contributors: Nat Hillard, Steve Leibman, Chris Pimlott, Michelle Park, Janell Schafer, Kelly Taylor